src/bindings.c

```diff
@@ -424,23 +426,25 @@ int pr_ipbind_create(server_rec *server, const pr_netaddr_t *addr,
   }
 
   i = ipbind_hash_addr(addr);
+  pr_trace_msg(trace_channel, 29, "hashed address '%s' to index %u",
+    pr_netaddr_get_ipstr(addr), i);
 
   /* Make sure the address is not already in use */
   for (ipbind = ipbind_table[i]; ipbind; ipbind = ipbind->ib_next) {
     if (pr_netaddr_cmp(ipbind->ib_addr, addr) == 0 &&
         ipbind->ib_port == port) {
-
-      /* An ipbind already exists for this IP address */
-      pr_log_pri(PR_LOG_WARNING, "notice: '%s' (%s:%u) already bound to '%s'",
-        server->ServerName, pr_netaddr_get_ipstr(addr), port,
-        ipbind->ib_server->ServerName);
-
-      errno = EADDRINUSE;
```

Comment:
Rather than erroring out with `EADDRINUSE` here (which prevented proper associating of different namebased vhosts), we needed to continue on.

I may need to refine this, so that we _do_ error out with `EADDRINUSE` _if_ the given vhost config does not have any associated names/ServerAlias.